### ISSUE TITLE
Misleading translation contexts may lead to unusable menus/dialogs

### ISSUE THREAD

**Issue by maoschanz** [maoschanz/drawing#353]
## Description

Example:

```
#. Context: answer to "where do you want to open the image?"
#: data/com.github.maoschanz.drawing.desktop.in src/ui/app-menus.ui
#: src/ui/shortcuts.ui src/ui/win-menus.ui src/window.py
msgid "New Window"
msgstr "新しいウィンドウ"
```
the context is valid for this version of the string in window.py, but NOT AT ALL for all the other ones!

**All** those contexts should be rewritten i think

## System

- App version: 0.7.0+
